File: Python/ExampleTests/production/ShoppingCart.py
class ShoppingCart:
    def __init__(self, items: str = =EmptyCart, shipping: str = $0, discount: str = $0, total_price: str = $0, shipping_address: str = =NoAddress, billing_address: str = =NoAddress):
        self.items = items
        self.shipping = shipping
        self.discount = discount
        self.total_price = total_price
        self.shipping_address = shipping_address
        self.billing_address = billing_address

File: TypeScript/ExampleTests/production/ShoppingCart.ts
export class ShoppingCart {
  items: OrderItemCollection;
  shipping: Dollar;
  discount: Dollar;
  totalPrice: Dollar;
  shippingAddress: Address;
  billingAddress: Address;

  constructor(items: OrderItemCollection = =EmptyCart, shipping: Dollar = $0, discount: Dollar = $0, totalPrice: Dollar = $0, shippingAddress: Address = =NoAddress, billingAddress: Address = =NoAddress) {
    this.items = items;
    this.shipping = shipping;
    this.discount = discount;
    this.totalPrice = totalPrice;
    this.shippingAddress = shippingAddress;
    this.billingAddress = billingAddress;
  }
}
